packages/block-editor/src/components/block-list/use-block-props/use-event-handlers.js

```diff
@@ -26,16 +26,16 @@ import { store as blockEditorStore } from '../../../store';
  */
 export function useEventHandlers( clientId ) {
 	const onSelectionStart = useContext( SelectionStart );
-	const { isSelected, rootClientId, index } = useSelect(
+	const { isBlockSelected, rootClientId, index } = useSelect(
 		( select ) => {
 			const {
-				isBlockSelected,
+				isBlockSelected: _isBlockSelected,
 				getBlockRootClientId,
 				getBlockIndex,
 			} = select( blockEditorStore );
 
 			return {
-				isSelected: isBlockSelected( clientId ),
+				isBlockSelected: _isBlockSelected,
 				rootClientId: getBlockRootClientId( clientId ),
 				index: getBlockIndex( clientId ),
```

Comment:
For all of these selectors, we could really use #31078. All of the results are only used in callbacks so it's pointless to call them here.